lib/Db/Helper.php

```diff
@@ -21,7 +21,26 @@
 
 class Helper
 {
+    public static function upsert(ConnectionInterface|\Doctrine\DBAL\Connection $connection, string $table, array $data, array $keys, bool $quoteIdentifiers = true): int|string
```

Comment:
What about some PhpDoc describing what to pass as `$data` and `$keys` arguments? It is not quite clear why there are two parameters and how they relate to each other.

Reply:
Good idea, I added a short description for both parameters.